torch/_dynamo/variables/misc.py

```diff
@@ -852,12 +852,16 @@ def call_function(
             self.value is functools.wraps
             and not kwargs
             and len(args) == 1
-            and args[0].source
+            and (args[0].source is not None or args[0].can_reconstruct(tx.output.root_tx))
```

Comment:
Just curious, does it need to be the root_tx?

Reply:
I think just tx is fine right?

Reply:
I get some assertions when it is not the root tx. The moral reason we're using the root tx is because one only reconstructs VT at the root tx (instead of InliningInstructionTranslators)